server/statshouse/add-metrics-batch.h

```diff
@@ -28,3 +28,5 @@ struct StatsHouseAddMetricsBatch {
 };
 
 StatsHouseMetric make_statshouse_value_metric(std::string name, double value, const std::vector<std::pair<std::string, std::string>> &tags);
+
+StatsHouseMetric make_statshouse_value_metrics(std::string name, std::vector<double>&& value, const std::vector<std::pair<std::string, std::string>> &tags);
```

Comment:
```suggestion
StatsHouseMetric make_statshouse_value_metrics(std::string &&name, std::vector<double> &&value, const std::vector<std::pair<std::string, std::string>> &tags);
```
Now it's copied always.